src/index.js

```diff
@@ -1,7 +1,8 @@
 import './style.css';
 import {
-  form, addTask, renderList, clearCompletedTasks,
-} from './modules/utilities.js';
+  addTask, renderList, clearCompletedTasks,
+} from './modules/methods.js';
+import { form } from './modules/variables.js';
 
 renderList();
 clearCompletedTasks();
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md)for this project, "`Update items object's value for completed key upon user actions, Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).`", Make sure when the `checks` as particular checkbox or marks a task as completed, it should be updated in the local storage. Please kindly fix that.